src/test/regress/sql/citus_local_table_triggers_mx.sql

```diff
@@ -0,0 +1,67 @@
+\set VERBOSITY terse
+
+SET citus.next_shard_id TO 1508000;
+SET citus.shard_replication_factor TO 1;
+SET citus.enable_local_execution TO ON;
+SET citus.log_local_commands TO ON;
+
+CREATE SCHEMA citus_local_table_triggers_mx;
+SET search_path TO citus_local_table_triggers_mx;
+
+-- ensure that coordinator is added to pg_dist_node
+SET client_min_messages to ERROR;
+SELECT 1 FROM master_add_node('localhost', :master_port, groupId => 0);
+RESET client_min_messages;
+
+CREATE TABLE citus_local_table (value int);
```

Comment:
can we use this file to exercise a foreign key test between citus local table and reference table on MX mode? Maybe  rename the file as well